src/Symfony/Bundle/TwigBundle/Extension/ActionsExtension.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  *
- * @deprecated Deprecated in 2.2, to be removed in 3.0.
+ * @deprecated since version 2.2, to be removed in 3.0.
```

Comment:
@stof @fabpot is the class still considered deprecated or only the construct method arguments?